examples/mlserver/models/text-classification-model/models.py

```diff
@@ -0,0 +1,19 @@
+from mlserver import MLModel
```

Comment:
this is great, love that it works out of the box - let's throw in the serving command as a comment just for convenience